app/models/user.rb

```diff
@@ -2,15 +2,28 @@
 #
 # Table name: users
 #
-#  id         :integer          not null, primary key
-#  height     :integer
-#  password   :string
-#  username   :string
-#  weight     :integer
-#  created_at :datetime         not null
-#  updated_at :datetime         not null
+#  id                     :integer          not null, primary key
+#  email                  :string           default(""), not null
+#  encrypted_password     :string           default(""), not null
+#  height                 :integer
+#  password               :string
+#  remember_created_at    :datetime
+#  reset_password_sent_at :datetime
+#  reset_password_token   :string
+#  username               :string
+#  weight                 :integer
+#  created_at             :datetime         not null
+#  updated_at             :datetime         not null
+#
+# Indexes
+#
+#  index_users_on_email                 (email) UNIQUE
+#  index_users_on_reset_password_token  (reset_password_token) UNIQUE
 #
 class User < ApplicationRecord
-  validates :username, presence: true
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
   validates :password, presence: true
```

Comment:
I don't think this validation is necessary— I believe Devise automatically validates the password by default (technically the `encrypted_password` since we never store the actual password).